parts/linux/cloud-init/artifacts/configure_azure0.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+# Check if the azure cni config is there... no need to run this script if not
+# Also don't want to run this when not using azure-cni
+[ ! -f /etc/cni/net.d/10-azure.conflist ] && exit 0
+
+# CNI team mentions that this is not needed for calico network policy to run this script
+if [[ "${NETWORK_POLICY}" == "calico" ]]; then
```

Comment:
if [[ "{{GetParameter "networkPolicy"}}" == "calico" ]]; then

Reply:
sorry , GetParameter won't work currently. it needs some refactoring. But I have one workaround. see my other comment